test/driverMetadata.test.mjs

```diff
@@ -28,12 +34,16 @@ describe('DriverMetadata', () => {
     Cache.reset();
   });
 
-
```

Comment:
Add this line

Reply:
Why? this is causing lint error

Reply:
I thought it was removed mistakenly. Ignore in that case